src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -162,30 +177,30 @@ public void testInsertWhenDataSetHasExtraColumns() throws SQLException {
     public void testInsertWhenTypeMismatch() throws SQLException {
         DataSet dataSet = createDataSet(new String[]{"typeMismatch", "name1", "25"});
         createTableWithData(TABLE_NAME);
-        databaseManager.connect(dbName, USER, PASSWORD);
+        databaseManager.connect(NAME_DB_TEST, LOGIN_FOR_TEST, PASSWORD_FOR_TEST);
         boolean actual = databaseManager.insert(TABLE_NAME, dataSet);
         assertFalse(actual);
     }
 
     @Test
     public void testDeleteWhenValidData() throws SQLException {
         createTableWithData(TABLE_NAME);
-        databaseManager.connect(dbName, USER, PASSWORD);
+        databaseManager.connect(NAME_DB_TEST, LOGIN_FOR_TEST, PASSWORD_FOR_TEST);
         boolean actual = databaseManager.delete(TABLE_NAME, 1);
         executeQuery(String.format("DROP TABLE IF EXISTS %s", TABLE_NAME));
```

Comment:
You drop table twice: in the method and in the @after

Reply:
good remark. I agree

Reply:
So it is. :)
I will fix it.